### ISSUE TITLE
Add hover tooltip on download button in Playground

### ISSUE THREAD

**Issue by danieljancar** [angular/angular#54619]
### Describe the problem that you experienced

As we know the cool angular.dev playground has a feature to download the provided/written source code. 
I find that at the first time of seeing/using this button it was pretty unclear to me what it does. First I thought it will just download the playground file I have currently open.

### Enter the URL of the topic with the problem

https://angular.dev/playground

### Describe what you want to experience that would fix the problem

I suggest a tooltip below the button, that displays when hovering over the button, which says `Download the current application source code` or similar.

https://material.angular.io/components/tooltip/overview

### Add a screenshot if that helps illustrate the problem

<img width="919" alt="image" src="https://github.com/angular/angular/assets/112062588/8c39ee4b-59bf-41a9-92bb-9bfbb50b745b">

### Provide any additional information here in as much as detail as you can

I'd be happy to implement this UX/Accessibility improvement. 😄

**Comment by nileshkr9919** [comment]
Hey.. I would like to work on it.

**Comment by danieljancar** [comment]
Hey @nileshkr9919,
Already working on a branch, feel free to create your own implementation. Interested on your solution. 🙈

**Comment by Ashwin-George** [comment]
I want to try solve the issue

**Comment by madrizr** [comment]
Hello! I want to try to solve the issue

**Comment by Sathish36311** [comment]
Shall I take this issue @danieljancar

**Comment by danieljancar** [comment]
Hey @Sathish36311 
Currently I'm working on other projects. Sure if you'd like! 🙈 👍🏻

**Comment by Sathish36311** [comment]
> Hey @Sathish36311 Currently I'm working on other projects. Sure if you'd like! 🙈 👍🏻
Hi @danieljancar will you please share the issue/repository you are currently working in, I can contribute something

**Comment by danieljancar** [comment]
There's no real progress anymore. Feel free to fork and solve the issue on your own. 😄

**Comment by Sathish36311** [comment]
Thank you @danieljancar 😊

**Comment by LeoSantich** [comment]
It looks like this issue is just waiting for a merge of #54634.  Kindly reply here if that is not the case for whatever reason and this issue still needs work.  Thank you.

**Comment by danieljancar** [comment]
Hey @LeoSantich 
The PR is still draft and would need some feedback on how it should be styled as there's no other use case yet, where such a tooltip was applied. Maybe considering globalising it or making it a shared component? Currently working on a whole lot of other stuff, so might take a while, if someone else wants to continue working on my progress...
Thanks for your awareness tho! 😄💯

**Comment by LeoSantich** [comment]
Thanks for replying, @danieljancar 
Has a tooltip been implemented on the download button in the draft, and you're waiting on feedback to see if it's good or not?  Or are you waiting on general styling feedback, i.e. how the rest of the project styles the tooltips?  If this is just adding a tooltip to a button, I may be able to help.
